src/gluonts/ev/metric_helpers.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Let's rename this file. I don't think these are helpers but maybe `stats` or something similar works better.